pyEpiabm/pyEpiabm/sweep/place_sweep.py

```diff
@@ -57,6 +57,18 @@ def __call__(self, time: float):
                             # Increment the infector's
                             # secondary_infections_count
                             infector.increment_secondary_infections()
+                            # Set the time between infector's infection time
+                            # and the infectee's exposure time (current time)
+                            # to be the exposure period of the infectee
+                            inf_to_exposed = \
+                                (time - infector.infection_start_times[-1])
+                            infectee.set_exposure_period(inf_to_exposed)
+                            # Finally, store the infector's latency period
+                            # within the infectee, which will be used in
+                            # calculating the generation_time
+                            (infectee
+                             .set_infector_latent_period(infector
+                                                         .latent_period))
```

Comment:
I don't love that this code is repeated in each sweep, and would rather it was handled in the queue sweep (or an infection sweep) to avoid repetition. If this code has to be in this file (i.e. because the infector is only known in these sweeps, as I can't remember), then perhaps consider a separate function, that might be a method of the abstract sweep class, which each individual sweep class can call.

Reply:
Ah yes, that makes sense, will work on that!